avocado/utils/linux_modules.py

```diff
@@ -208,34 +208,107 @@ def get_loaded_modules():
         return [astring.to_text(_.split(b' ', 1)[0]) for _ in proc_modules]
 
 
-def check_kernel_config(config_name):
+def check_kernel_config(config_name, session=None):
     """
     Reports the configuration of $config_name of the current kernel
 
     :param config_name: Name of kernel config to search
+    :param session: guest session, command is run on host if None
     :type config_name: str
     :return: Config status in running kernel (NOT_SET, BUILTIN, MODULE)
     :rtype: :class:`ModuleConfig`
     """
 
-    kernel_version = platform.uname()[2]
+    def check_local_kernel_config(config_name):
+        """
+        Reports the configuration of $config_name of the current kernel
+
+        :param config_name: Name of kernel config to search
+        :return: Config status in running kernel (NOT_SET, BUILTIN, MODULE)
+        :rtype: :class:`ModuleConfig`
+        """
+        kernel_version = platform.uname()[2]
+
+        config_file = '/boot/config-' + kernel_version
+        with open(config_file, 'r') as kernel_config:  # pylint: disable=W1514
+            for line in kernel_config:
+                line = line.split('=')
+
+                if len(line) != 2:
```

Comment:
why continue for only `len(line) != 2`?
or is this something to be `len(line) <= 2`
just curious to know the logic here.